global_helpers/panther_oss_helpers.py

```diff
@@ -412,10 +412,11 @@ def add_parse_delay(event, context: dict) -> dict:
 
 # check for presence of user id in KV store for the purpose of modifying severity or suppressing
 # alerts based on expected actions for a new user
+# returns as string instead of a BOOL until it is possible to mock Boolean values in PAT
```

Comment:
i would make this return a boolean and convert it in the rule. It will make updating individual rules easier because you'll be able to do them one at a time (presumably by changing some parameter in the .yml). Otherwise, if you change this helper in the future (from string back to bool), you'll have to change every rule that depends on it.

If you still want to keep it as a string, maybe we should rename the function and the return value to "check identity" or even "check value". No other detections use this function right now, and here we're using it to look up an account ID or alias, not a "user".